pmd-cataloger/src/test/java/sfdc/sfdx/scanner/pmd/catalog/PmdCatalogJsonTest.java

```diff
@@ -1,75 +1,75 @@
 
-		// Execute
-		final JSONObject jsonObject = catalogJson.constructJson();
+        // Execute
+        final JSONObject jsonObject = catalogJson.constructJson();
 
-		// Verify
-		//[{"paths":["rule path"],"name":"rule name"}]
-		final String expectedRulesetJson = String.format("[{\"engine\":\"%s\",\"paths\":[\"%s\"],\"name\":\"%s\"}]",
-			PmdCatalogJson.PMD_ENGINE_NAME, RULE_PATH, RULE_NAME);
-		assertEquals(expectedRulesetJson, jsonObject.get(PmdCatalogJson.JSON_RULESETS).toString());
+        // Verify
+        //[{"paths":["rule path"],"name":"rule name"}]
+        final String expectedRulesetJson = String.format("[{\"engine\":\"%s\",\"paths\":[\"%s\"],\"name\":\"%s\"}]",
+            PmdCatalogJson.PMD_ENGINE_NAME, RULE_PATH, RULE_NAME);
+        assertEquals(expectedRulesetJson, jsonObject.get(PmdCatalogJson.JSON_RULESETS).toString());
 
-		final String expectedCategoryJson = String.format("[{\"engine\":\"%s\",\"paths\":[\"%s\"],\"name\":\"%s\"}]",
-			PmdCatalogJson.PMD_ENGINE_NAME, CATEGORY_PATH, CATEGORY_NAME);
-		assertEquals(expectedCategoryJson, jsonObject.get(PmdCatalogJson.JSON_CATEGORIES).toString());
+        final String expectedCategoryJson = String.format("[{\"engine\":\"%s\",\"paths\":[\"%s\"],\"name\":\"%s\"}]",
+            PmdCatalogJson.PMD_ENGINE_NAME, CATEGORY_PATH, CATEGORY_NAME);
+        assertEquals(expectedCategoryJson, jsonObject.get(PmdCatalogJson.JSON_CATEGORIES).toString());
 
-		// Rules json has its own test where we verify the Json contents. Here, we only confirm that it exists
-		assertTrue("JSON should contain 'rules' element", jsonObject.containsKey(PmdCatalogJson.JSON_RULES));
-	}
+        // Rules json has its own test where we verify the Json contents. Here, we only confirm that it exists
+        assertTrue(jsonObject.containsKey(PmdCatalogJson.JSON_RULES), "JSON should contain 'rules' element");
+    }
 
-	private PmdCatalogRule getPmdCatalogRuleMock() {
-		final PmdCatalogRule catalogRule = mock(PmdCatalogRule.class);
-		final JSONObject jsonObject = mock(JSONObject.class);
+    private PmdCatalogRule getPmdCatalogRuleMock() {
+        final PmdCatalogRule catalogRule = mock(PmdCatalogRule.class);
+        final JSONObject jsonObject = mock(JSONObject.class);
 
-		doReturn(jsonObject).when(catalogRule).toJson();
+        doReturn(jsonObject).when(catalogRule).toJson();
 
-		return catalogRule;
-	}
+        return catalogRule;
+    }
 
-	private PmdCatalogCategory getPmdCatalogCategoryMock(String name, String path) {
-		final PmdCatalogCategory category = mock(PmdCatalogCategory.class);
+    private PmdCatalogCategory getPmdCatalogCategoryMock(String name, String path) {
+        final PmdCatalogCategory category = mock(PmdCatalogCategory.class);
 
-		doReturn(name).when(category).getName();
-		doReturn(path).when(category).getPath();
+        doReturn(name).when(category).getName();
+        doReturn(path).when(category).getPath();
 
-		return category;
-	}
+        return category;
+    }
```

Comment:
This review looks more noisy than it should simply because the files used to have a bunch of tab characters in them it seems. Intellij automatically updates my files to replace tabs with spaces. I'd recommend everyone on the team should do the same.